src/controllers/util/index.ts

```diff
@@ -1,15 +1,23 @@
 import { Context } from 'koa';
 import { API_VERSION } from '../../routes/utils/constants';
 import { isHttpStatusSuccess } from '../../v0/util';
+import { HttpStatus } from '../../v0/util/constant';
 
 export default class ControllerUtility {
+  private static handleForNotfoundStatus(status: number) {
```

Comment:
make this function generic to handle other status codes

Reply:
Done.